### DISCUSSION TITLE
Teammate name color is red for respawning campaign player

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14927] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In campaign mode while you are respawning other teammates name colors are red

### Reproduction steps

1. join a multiplayer campaign
2. during waiting for respawning, teammates name color is red

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by NilanthAnimosus** [comment]
Closing as fixed and merged. Should be fixed now in a future build (Likely in a future hotfix)
